polkadot/zombienet_tests/functional/0006-dispute-freshly-finalized.toml

```diff
@@ -0,0 +1,45 @@
+[settings]
+timeout = 1000
+
+[relaychain.genesis.runtime.configuration.config]
+  max_validators_per_core = 2
+  needed_approvals = 3
+
+[relaychain]
+default_image = "{{ZOMBIENET_INTEGRATION_TEST_IMAGE}}"
+chain = "rococo-local"
+default_command = "polkadot"
+
+[relaychain.default_resources]
+limits = { memory = "4G", cpu = "2" }
+requests = { memory = "2G", cpu = "1" }
+
+  [[relaychain.node_groups]]
+  name = "honest"
+  count = 12
+  args = ["-lparachain=debug"]
+
+  [[relaychain.nodes]]
+  image = "{{MALUS_IMAGE}}"
+  name = "malus"
+  command = "malus dispute-finalized-candidates"
+  args = [ "--alice", "-lparachain=debug,MALUS=trace", "--dispute-offset=3" ]
```

Comment:
Out of curiosity - why `dispute-offset=3` and not `1` for example?

Reply:
Arbitrary parameter. Wanted to test something that's finalized for longer just to make sure it properly registers a dispute, but technically 1 would be just as good.